### PR TITLE
Update README.md with gemspec

### PR DESCRIPTION
**Pull request by dhivyada**
Updates README.md

### CODE REVIEW

**Comment by dhivyada** [comment]
Thank you for the feedback @jethrodaniel. Have updated the README as per your suggestions. Will raise separate PRs for the rest of the changes

File: README.rdoc [hunk 1]
```diff
@@ -47,10 +47,23 @@ useful. If there is another library which handles it, we list it here.
 
 == Develop
 
+* Clone the repository using
+  git clone https://github.com/ruby-ldap/ruby-net-ldap
```
Comment (jethrodaniel) [inline README.rdoc:51]: Need line breaks after `* Clone the repository using`  to ensure the code is formatted, not stuck to the end of the line.

You can preview it with Github's `view file` option: https://github.com/ruby-ldap/ruby-net-ldap/blob/b1871a0af286f941ea13769e03a9500110d0a76d/README.rdoc

Same goes for the below

Comment (dhivyada) [reply README.rdoc:52]: Thank you. formatting done :)

Comment (jethrodaniel) [reply README.rdoc:52]: image

If you remove the bullets, it'll match the existing code blocks

```
To install all the dependencies for local development:

  gem install bundler # if you don't have it
  bundle install

This task will run the test suite and the
```

**Comment by jethrodaniel** [comment]
:+1:  thanks

File: lib/net/ldap.rb [hunk 2]
```diff
@@ -1315,6 +1316,7 @@ def new_connection
       :hosts                   => @hosts,
       :encryption              => @encryption,
       :instrumentation_service => @instrumentation_service,
+      :socket_class            => @socket_class,
```
Comment (HarlemSquirrel) [inline lib/net/ldap.rb:1319]: Changes to this file should be in a separate PR.

Comment (HarlemSquirrel) [reply lib/net/ldap.rb:1319]: @dhivyada :point_up:

**Comment by HarlemSquirrel** [comment]
Closing as stale. Please open a new PR if there' something else you'd like to add. Thank you!
